src/composables/useFormSpaceProposal.ts

```diff
@@ -51,7 +51,7 @@ const userSelectedDateStart = ref(false);
 const userSelectedDateEnd = ref(false);
 const sourceProposalLoaded = ref(false);
 
-export function useFormSpaceProposal() {
+export function useFormSpaceProposal({ spaceType = 'default' } = {}) {
```

Comment:
```suggestion
export function useFormSpaceProposal({ spaceType = 'default' }) {
```
Doesn't seem to need `= {}` as `spaceType` is always defined